cfbs.json

```diff
@@ -42,7 +42,7 @@
     },
     "inventory-etc-hosts": {
       "description": "Inventory entries from /etc/hosts",
-      "tags": ["inventory", "experimental"],
+      "tags": ["inventory", "supported"],
```

Comment:
Please put it as the first tag here as well

Reply:
i did